data/config/toolbox.conf

```diff
@@ -0,0 +1,13 @@
+[general]
```

Comment:
We need a `meson.build` to actually install this file, so that users can tweak it easily.